File: app/src/main/java/com/dergoogler/liquidwars/activities/InstructionsActivity.java


package com.dergoogler.liquidwars.activities;

import android.os.Bundle;
import android.widget.TextView;
import android.view.Window;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.view.WindowCompat;

import com.dergoogler.liquidwars.R;

public class InstructionsActivity extends LiquidCompatActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.instructions);
    }
}

File: app/src/main/java/com/dergoogler/liquidwars/activities/InstructionsActivity.kt
package com.dergoogler.liquidwars.activities

import android.os.Bundle
import com.dergoogler.liquidwars.R

class InstructionsActivity : LiquidCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.instructions)
        setAdsBanner(R.id.instructions_ads_banner)
    }
}
